### PR TITLE
User can see a charts showing time spent on projects and money earned

### PR DESCRIPTION
**Pull request by LauraRe**
#### Description
_As a User
In order to be able to track my progress
I would like to see a summary chart on the dashboard with total time spent/money gained on projects_

Changes proposed in this pull request:

* add chart showing time spent on projects per month (current year)
* add chart showing money gained from projects per month (current year)
* add box showing time spent on project (today)
* add box showing revenue (today)

#### What I have learned working on this feature: 
* how to display data in charts and boxes
* how to modify labels and legend in react-chartjs-2

#### Screenshots:
image

### CODE REVIEW

File: src/components/timeTrackingTable.js [hunk 1]
```diff
@@ -239,91 +239,96 @@ export class TimeTrackingTable extends Component {
...
-							<Table.HeaderCell>Customer</Table.HeaderCell>
-							<Table.HeaderCell>Project</Table.HeaderCell>
-							<Table.HeaderCell>Task</Table.HeaderCell>
-							<Table.HeaderCell> </Table.HeaderCell>
-						</Table.Row>
-					</Table.Header>
+				<Segment name="timetracking">
+					<Header as="h1" textAlign="center">
+						Time Tracking
+				</Header>
```
Comment (perhegg) [inline src/components/timeTrackingTable.js]: indentation

Comment (LauraRe) [reply src/components/timeTrackingTable.js]: fixed

**Comment by LauraRe** [comment]
fixed

File: cypress/integration/displayCharts.feature.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  beforeEach(function () {
+		cy
+			.get('button').contains('Get started here!').click()
+			.get(':nth-child(1) > .ui > input').type('susan_super')
+			.get(':nth-child(2) > .ui > input').type('api_kitten')
+			.get('button').contains('Login').click()
+	})
+
+  it('User can see saved times', () => {
+	cy
```
Comment (vick3d) [inline cypress/integration/displayCharts.feature.js]: indentation

Comment (LauraRe) [reply cypress/integration/displayCharts.feature.js]: done

**Pull request merged**